### ISSUE TITLE
Création du fichier initial nomenclatures_taxonomie_inpn_data.sql

### ISSUE THREAD

**Issue by Adrien-Pajot** [PnX-SI/GeoNature#1912]
Slt, 

Initialement, comment a été construit le fichier suivant : 
https://github.com/PnX-SI/Nomenclature-api-module/blob/master/src/pypnnomenclature/migrations/data/nomenclatures_taxonomie_inpn_data.sql

Sommes-nous sûr du report de chaque nomenclature pour chaque groupe ? Ceci fait suite à une question du PNR de la Forêt d'Orient sur la possibilité de mettre juvénile à un Amphibien. 

Personnellement, je ne sais pas si caractériser un Amphibien comme juvénile est juste ou faux, mais là n'est pas la question :)

**Comment by DonovanMaillard** [comment]
C'est une construction qui est basée sur un tri de naturalistes, qui ont trié par groupe inpn ce qui semblait pertinent ou non à leur avis. Ce travail a été fait en bonne partie par Gil et un peu par moi pour compléter/affiner. Il laisse donc indéniablement de la place à des erreurs, approximations etc. D'une manière générale, quand on était pas trop surs de notre coup, on a fait le choix de laisser trop de possibilités plutôt que trop peu. Un juvénile selon la définition peut tout à fait s'utiliser pour un amphibien meme si c'est redondant avec têtard, subadulte etc, mais comme c'est pas forcément faux il a du être laissé. 

Ces correspondances sont encore moins idéales sur des groupes méconnus, et ne sont pas faites sur les nomenclatures les plus récentes comme le comportement. Ca peut donc être complété, rectifié si c'est générique (là pour les juvéniles ca me semble pas pertinent de le retirer pour tout le monde par exemple, mais ce n'est que mon avis :)

Voilà donc pour le coté "historique", c'est quelque chose qui gagnerait à être fait au niveau des référentiels et pas au niveau de l'outil en soit, ça permettrait de définir une ligne de conduite et une sémantique unique

**Comment by gildeluermoz** [comment]
Je confirme ce travail a été fait à l'époque pour répondre au besoin et à l'usage qu'il trouve aujourd'hui dans geonature. Fait avec les moyens et le temps disponibles. Comme le dit Donovan, il est perfectible et mériterait le regard d'experts taxonomistes. Ça c'est pour sa forme à l'installation de geonature que l'on peut encore aujourd'hui considérer comme provisoire

Comme ce travail n'était qu'une ébauche et qu'il n'y a pas d'interface dans geonature admin pour le modifier ou pour le compléter, il est parfaitement envisageable de le modifier manuellement et directement dans la base de données pour répondre à un usage précis dans le cadre d'une instance de geonature.

**Comment by camillemonchicourt** [comment]
C'est à faire en effet que de relire tout ça et de compléter pour les groupes ou nouvelles nomenclatures.
Voir https://github.com/PnX-SI/GeoNature/issues/263

Chacun peut en effet les ajuster et affiner sur sa BDD, mais ça serait mieux de faire un truc global et de le contribuer/partager si il y a des manques et compléments à faire.

**Comment by Adrien-Pajot** [comment]
Super, ces réponses sont très claires et mènent à trois conclusions pour nous : 
1/ ceci n'est pas un standard 
2/ il est possible de changer pour chaque instance de GeoNature
3/ une centralisation au niveau de GeoNature (ou du SINP) doit être réfléchie 

Merci à tous pour votre réactivité

**Comment by camillemonchicourt** [comment]
Oui, à noter qu'on en avait parlé avec l'équipe de Patrinat et ce que l'on avait fait les intéressait vraiment.
Donc on peut imaginer que ça remonte à terme au niveau des ressources nationales.

**Comment by AudreyEnGuyane** [comment]
Bonjour à tous,
On est en train d'identifier quelques manques: du genre stade de vie "immature" pour les amphibiens, "en fleur" pour la flore...
Je fais des propositions directement dans le fichier sql avec un pull request?

Ensuite, mon botaniste a tiqué sur l'absence d'un stade de vie "Fleuri" alors que "Fané" existe. Comment font vos autres botanistes?

**Comment by DonovanMaillard** [comment]
> Bonjour à tous, On est en train d'identifier quelques manques: du genre stade de vie "immature" pour les amphibiens, "en fleur" pour la flore... 

Bonjour @AudreyEnGuyane , sauf erreur il y a 2 choses différentes ici. Le stade immature existe bien, il n'est simplement pas associé aux amphibiens : c'est bien un cas propre à GeoNature qui peut se gérer au niveau de l'outil. Le stade en fleur en revanche n'existe simplement pas dans le référentiel, et donc il est plus compliqué de le rentrer au niveau de geonature car on transfèrerait des données ayant des nomenclatures non standards...

**Comment by AudreyEnGuyane** [comment]
Ok pour le stade "immature". Je n'ai pas trouvé comment ça se gère dans l'outil: il faut passer par la base de données ?

Ce qui m’interpelle c'est de savoir comment font les botanistes qui font la saisie sans ce stade "en fleur"? Car on peut compter le nombre d'individus en fleur sans non plus compter les hampes florales... Quelqu'un peut m'éclairer? Vous pouvez vous renseigner avec vos agents de terrain?

Sinon, serait-il possible de m'indiquer la personne ressource à l'INPN pour faire codifier une nouvelle nomenclature STADE_VIE ?
Merci d'avance...

**Comment by camillemonchicourt** [comment]
Pour les stades de vie botanique, sujet déjà évoqué : https://github.com/PnX-SI/GeoNature/issues/636

Les nomenclatures botaniques sont en cours d'ajout côté SINP.

PS : Il vaut mieux aborder un sujet par ticket, sinon on ne sait plus de quel sujet on parle dans un ticket quand on mélange plusieurs sujets.
Et on ne sait plus quand on peut fermer un ticket ou non quand il aborde différents sujets.
